src/config.c

```diff
@@ -428,7 +428,7 @@ void config_set_defaults()
 				if (username) {
 					destValue->value_string = _strdup(username);
 				} else {
-					destValue->value_string = _strdup(language_get_string(STR_PLAYER_DEFAULT_NAME));
+					destValue->value_string = _strdup(language_get_string(STR_RIDE_COMPONENT_PLAYER_CAPITALISED));
```

Comment:
This is the default name when you start a multiplayer game. I wouldn't make it the same a the string for a mini golf player, it might not apply to all languages.